app/views/state_file/questions/eligibility_offboarding/_other_options.html.erb

```diff
@@ -1,30 +0,0 @@
-<div class="reveal">
```

Comment:
Ditching this file is an improvement!

Reply:
actually it turns out I was premature and it just wasn't caught by any tests, but this actually WAS being referenced from a second place. So I might put everything back in so everything isn't repeated. In any case, a fix and test is coming...